swift/Sources/Addresses/BitcoinAddressExtension.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+
+extension BitcoinAddress: Address, Equatable {
```

Comment:
Filename suggestion: `BitcoinAddress+Extension.swift`

Reply:
I like that pattern when the part after the `+` describes what the extension is for.  In this case it doesn't.